sql/item_weapon.sql

```diff
@@ -52,7 +52,7 @@ INSERT INTO `item_weapon` VALUES (16387,'poison_cesti',1,0,0,0,0,4,1,528,6,0);
 INSERT INTO `item_weapon` VALUES (16388,'himantes',1,0,0,0,0,4,1,528,7,0);
 INSERT INTO `item_weapon` VALUES (16389,'coeurl_cesti',1,0,0,0,0,4,1,528,8,0);
 INSERT INTO `item_weapon` VALUES (16390,'bronze_knuckles',1,0,0,0,0,4,1,576,5,0);
-INSERT INTO `item_weapon` VALUES (16391,'brass_knuckles',1,0,0,0,0,4,1,576,7,0);
```

Comment:
:confounded: 
both bronze and brass NQ's have same delay of `+96` the plus indicating it is added to the h2h base delay.

Reply:
this pr is just to darn big to even review all at once.